src/Microsoft.Diagnostics.TestHelpers/ProcessRunner.cs

```diff
@@ -403,10 +403,23 @@ private async Task<int> InternalWaitForExit(Task<Process> startProcessTask, Task
             },
             TaskCreationOptions.LongRunning);
 
-            DebugTrace("awaiting process {0} exit, stdOut, and stdErr", p.Id);
-            await Task.WhenAll(processExit, stdOutTask, stdErrTask);
+            DebugTrace("awaiting process {0} exit", p.Id);
+            await processExit;
             DebugTrace("await process {0} exit, stdOut, and stdErr complete", p.Id);
```

Comment:
Nit: this DebugTrace still says that stdOut and stdErr are completed where it is just the process now.